libs/ui/src/lib/useful-resources/useful-resources.tsx

```diff
@@ -0,0 +1,136 @@
+import CardGradient from '../card-gradient/card-gradient';
+import Image from 'next/image';
+import gumroad from '../../assets/gumroad-thumbnail.png';
+import bootcampLogo from '../../assets/bootcamp.png';
+import Button from '../button/button';
+import { ArrowTopRightOnSquareIcon } from '@heroicons/react/24/solid';
+import { Text } from '../text/text';
+import Gravatar from 'react-gravatar';
+
+export function UsefulResources() {
+  return (
+    <div>
+      <Text variant="h2" className="text-center">
+        Useful Resources
+      </Text>
+      <div className="flex flex-col justify-center gap-16 py-12 md:flex-row">
+        <CardGradient
```

Comment:
Instead of duplicating everything, refactor `CardHomepage`
- [ ] add a way to change the size props
- [ ] keep the same design / content